scripts/data/products.csv

```diff
@@ -851,8 +851,8 @@ cheese,sticks_string_cheese,sticks string cheese,sticks string cheeses,dairy,f,f
 cheese,stilton_cheese,stilton cheese,stilton cheeses,dairy,f,f,t,f,t
 cheese,swiss_cheese,swiss cheese,swiss cheeses,dairy,f,f,t,f,t
 cheese,velveeta_cheese,velveeta cheese,velveeta cheeses,dairy,f,f,t,f,t
-cheese,weight_chevre_style_goat_cheese,weight chevre-style goat cheese,weight chevre-style goat cheeses,dairy,f,f,t,f,t
```

Comment:
Note: some of these old/removed product identifiers (e.g. `weight_chevre_style_goat_cheese`) also exist in `product_nutrition.csv`.  Because the data quality of that file is generally considered low-quality at the moment, we do not use it, but it may still make sense to edit the corresponding identifiers there.

Reply:
I think it's probably better to remove the `product_nutrition.csv`-related database tables and functionality, and then to re-design that if-and-when we want that functionality in future.  I'll file a cleanup issue.